### PR TITLE
Subsetting Zilla Slab

### PR DESCRIPTION
**Pull request by tkadlec**
This PR addresses https://github.com/mdn/sprints/issues/254.

It introduces a subsetted version of the Zilla Slab font to be used instead of the full file, cutting the font file sizes in half.

### CODE REVIEW

**Comment by tkadlec** [comment]
@schalkneethling and @jwhitlock: This one is pretty lightweight, me thinks.

**Comment by tkadlec** [comment]
Ooo....almost forgot about the file I used to subset. I mentioned it in https://github.com/mdn/sprints/issues/254:

> I created a file that contains the unicode ranges to include. There are other shortcuts to go to certain subsets, but I like the clarity this gives us in case we edit the subset in the future

I wasn't sure exactly where to put it. Right now it's in `/kuma/static/`.

File: kuma/static/latin-subset [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+U+0100-017F
+
+# Latin Extended-B
+U+0180-024F
+
+# General Punctuation
+U+2000-206F
+
+# Currency Symbols
+U+20A0-20CF
```
Comment (schalkneethling) [inline kuma/static/latin-subset]: Nit: Missing empty line at end of file. This seems to be a new file? I am genuinely curious as to the purpose of this file? @tkadlec ?

**Comment by schalkneethling** [comment]
@tkadlec Thanks for this, how would you suggest we go about testing this? Which pages etc. did you test? Thanks!

File: kuma/static/latin-subset (hunk 1 above)
Comment (tkadlec) [reply kuma/static/latin-subset]: Added the new line. Also added some info to the top of the file for clarity. The file is used to generate the subset. You can see more information here: https://github.com/mdn/sprints/issues/254#issuecomment-427059741

**Comment by tkadlec** [comment]
@schalkneethling Good question! I tested a handful of random pages in each locale. Here are the pages I tested:

```
/
/docs/Web/JavaScript
/docs/Mozilla/Add-ons/WebExtensions
/docs/Web/HTML
/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach
```

And here are the locales:

```
Svenska (sv-SE)
Portugues (pt-PT) and (pt-BR)
Polski (pl)
Nederlands (nl)
Melayu (ms)
Taqbaylit (kab)
Italiano (it)
Bahasa Indonesia (id)
magyar (hu)
Francais (fr)
suomi (fi)
Espanol (es)
Deutsche (de)
Catala (ca)
English (en-US)
```

It's tedious, unfortunately. :/

**Comment by schalkneethling** [comment]
Thanks for the additional information @tkadlec - I will take this for a test run.

**Comment by schalkneethling** [comment]
@tkadlec I notice that the issue mentions subsetting ZillaSlab and OpenSans but this only address ZillaSlab. Is that intentional, perhaps to keep the work separate?

**Comment by schalkneethling** [comment]
@tkadlec I see there are still opensans that looks like it is being used for https://github.com/mozilla/kuma/tree/master/kuma/static/fonts/locales/ro

**Comment by tkadlec** [comment]
Ah, yeah. And that locale is being killed per
https://github.com/mdn/sprints/issues/449

**Comment by jwhitlock** [comment]
No, that review works for me.

**Comment by jwhitlock** [comment]
I can't find a relevant existing bug, I'll write a new one. I can add a bug # and squash on merge.

**Pull request merged**